patterns/hybris/jolla-hw-adaptation-hybris.yaml

```diff
@@ -1,18 +1,9 @@
 Description: Pattern with packages for the generic Hybris HW Adaptation
 Name: jolla-hw-adaptation-hybris
```

Comment:
I don't really see why this is separate pattern, we have some hybris related stuff in jolla-hw-adaptation-* already. I would add all the leftovers to jolla-hw-adaptation and drop this pattern.